crypto/crypto_entropy.c

```diff
@@ -37,6 +39,21 @@ static void update(uint8_t *, size_t);
 static int reseed(void);
 static void generate(uint8_t *, size_t);
 
+#ifdef CPUSUPPORT_X86_RDRAND
+static void
+update_from_rdrand() {
```

Comment:
`void` should go in here.  Functions without parameter lists is an obsolescent feature of the C language.